src/classes/BDI_FieldMappingCustomMetadata.cls

```diff
@@ -134,15 +134,15 @@ public with sharing class BDI_FieldMappingCustomMetadata implements BDI_FieldMap
             //Retreive the Data Import Settings to determine the default field mapping set.
             Data_Import_Settings__c dis = UTIL_CustomSettingsFacade.getDataImportSettings();
 
-            fieldMappingSetName = UTIL_Namespace.adjustNPSPNamespace(dis.Default_Data_Import_Field_Mapping_Set__c);
+            fieldMappingSetName = UTIL_Namespace.alignSchemaNSWithEnvironment(dis.Default_Data_Import_Field_Mapping_Set__c);
         }
 
         return fieldMappingSetName;
     }
 
     private void retrieveCustMetadata(){
         //If the code is not in the npsp namespace, then the npsp field prefixes will need to be systematically removed.
-        Boolean removeNPSPNamespace = (UTIL_Namespace.getNamespace() != 'npsp');
+        Boolean removeNPSPNamespace = (UTIL_Namespace.getNamespace() != UTIL_Namespace.NPSP_NS);
```

Comment:
@force2b I'll go ahead and make this change, although I think it should go in UTIL_Namespace since that is where all the related methods are.  Also, other pieces of code may need similar functionality in the future (ie other custom metadata where we need to specify a field or object name).